src/components/SignInEmail.js

```diff
@@ -0,0 +1,35 @@
+//TODO: create a email input
+import React from "react"
+import { Form, FormGroup, Label, Input, NavLink } from "reactstrap"
+
+export default class SignInEmail extends React.Component {
+  render() {
+    return (
+      <Form inline>
+        <FormGroup>
+          <Label for="exampleEmail" hidden>
```

Comment:
example means example, you found it in example code, you copy and paste this code, then you HAVE TO change everything called example
BECAUSE it is not an example anymore, it's your project code